src/behaviors/localization/locales/es.ts

```diff
@@ -24,7 +24,7 @@ const es:IPartialLocaleValues = {
         ],
         formats: {
             time: "HH:mm",
-            datetime: "D MMMM [de] YYYY HH:mm",
+            datetime: "D [de] MMMM [de] YYYY HH:mm",
```

Comment:
Yes, I am sure. Further reference [here](http://lema.rae.es/dpd/srv/search?id=7Yloy2M3SD6pcC3Zyx) (in Spanish sorry) The website belongs to the _Spanish Royal Academy_. Which is the authority that legislates the language (yes, we are cool as that :P).